### PR TITLE
Add PRR questions to node-role TOC

### PR DESCRIPTION
**Pull request by smarterclayton**
/assign @liggitt

### CODE REVIEW

**Comment by wojtek-t** [comment]
/lgtm

@smarterclayton - you can add me as PRR approver to the kep.yaml:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L16

**Comment by smarterclayton** [comment]
Updated with implementation history and added @wojtek-t to approvers for PRR

File: keps/sig-architecture/2019-07-16-node-role-label-use.md [hunk 1]
```diff
@@ -15,9 +15,10 @@ reviewers:
 approvers:
   - "@thockin"
   - "@derekwaynecarr"
+  - "@wojtek-t"
 creation-date: 2019-07-16
-last-updated: 2019-07-16
-status: implementable
+last-updated: 2020-06-01
+status: implemented
```
Comment (palnabarun) [inline keps/sig-architecture/2019-07-16-node-role-label-use.md:21]: @smarterclayton -- KEPs are marked `implemented` when they graduate to `Stable`. On the basis of https://github.com/kubernetes/enhancements/issues/1143#issuecomment-613159084, the KEP should have the `implementable` status.

File: keps/sig-architecture/2019-07-16-node-role-label-use.md [hunk 2]
```diff
@@ -15,9 +15,10 @@ reviewers:
 approvers:
   - "@thockin"
   - "@derekwaynecarr"
+  - "@wojtek-t"
```
Comment (wojtek-t) [inline keps/sig-architecture/2019-07-16-node-role-label-use.md:18]: in the meantime, we have added a dedicated `prr-approvers` field for it:
https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml#L17

[sorry - this process is evolving as we speak]

**Comment by smarterclayton** [comment]
Updated

**Comment by wojtek-t** [comment]
/lgtm

Thanks Clayton!

**Pull request merged**